pkg/action/install.go

```diff
@@ -716,5 +724,10 @@ func (c *ChartPathOptions) LocateChart(name string, settings *cli.EnvSettings) (
 	if version != "" {
 		atVersion = fmt.Sprintf(" at version %q", version)
 	}
-	return filename, errors.Errorf("failed to download %q%s (hint: running `helm repo update` may help)", name, atVersion)
+
+	hint := "running `helm repo update` may help"
+	if strings.HasPrefix(name, "oci://") {
+		hint = "running `helm registry login` may help)"
```

Comment:
Same... no comments about CLI in pkg/action

Reply:
resolved in 3e17c1a7efe7396b6c9cda5dbdb1a46a759e4b91